src/scrape.ts

```diff
@@ -360,7 +360,7 @@ async function get_recent(session: Session): Promise<Recent> {
   const recentActivityArray =
     [...document.querySelectorAll("gradebookScore")].map(x =>
       Object.fromEntries([
-        "date", "classname", "score", "assignment",
+        "date", "classname", "grade", "assignmentname",
```

Comment:
This change, along with the associated change on the frontend, causes unneeded API breakage. The names might not be ideal, but it's better just to leave them as is, so that we can avoid incompatibilities.